pkg/test/framework/components/ingress/ingress.go

```diff
@@ -21,6 +21,15 @@ import (
 	"istio.io/istio/pkg/test/framework/resource"
 )
 
+// IgType defines ingress gateway type
+type IgType int
```

Comment:
Can we change to more readable type name? `IngressTLSType` or something else?

Reply:
Changed to IngressGatewayType.